cmake/dependencies.cmake

```diff
@@ -278,7 +278,10 @@ if(ENABLE_OPENCV)
             elseif(LINUX_OS_NAME STREQUAL "Ubuntu 18.04" AND X86_64)
                 set(OPENCV_SUFFIX "ubuntu18")
                 set(OPENCV_HASH "db087dfd412eedb8161636ec083ada85ff278109948d1d62a06b0f52e1f04202")
-            elseif((LINUX_OS_NAME STREQUAL "Ubuntu 20.04" OR LINUX_OS_NAME STREQUAL "LinuxMint 20.1") AND X86_64)
+            elseif((LINUX_OS_NAME STREQUAL "Ubuntu 20.04" OR
+                    LINUX_OS_NAME STREQUAL "Ubuntu 21.10" OR
+                    LINUX_OS_NAME STREQUAL "Ubuntu 22.04" OR
+                    LINUX_OS_NAME STREQUAL "LinuxMint 20.1") AND X86_64)
                 set(OPENCV_SUFFIX "ubuntu20")
```

Comment:
Should we update suffix?

Reply:
we have to update packages on 01org then, but it's problematic now..